js/interface.js

```diff
@@ -139,7 +143,7 @@ switch (data.type) {
 }
 
 // Open data source query provider inline
-var dsQueryProvider = Fliplet.Widget.open('com.fliplet.data-source-query', {
+var dsQueryProvider = Fliplet.Widget.open('com.fliplet.data-source-query-andrey', {
```

Comment:
@upplabs-alex-levchenko Please explain why we are asked to review the PR when it contains this error?

Reply:
@tonytlwu Sometimes even I make mistakes, didn't you? :)